pkg/agent/cniserver/ipam/ipam_delegator.go

```diff
@@ -36,6 +36,9 @@ type IPAMDelegator struct {
 	pluginType string
 }
 
+var execPluginWithResultFunc = invoke.ExecPluginWithResult
```

Comment:
Use this format?

```
var (
       execPluginWithResultFunc = invoke.ExecPluginWithResult
       execPluginNoResultFunc = invoke.ExecPluginWithoutResult
)
```

Maybe also add a comment to explain that declaring these variables for test?

Reply:
Done.